src/Standards/BestIt/Sniffs/TypeHints/ReturnTypeDeclarationSniff.php

```diff
@@ -173,6 +174,18 @@ protected function fixDefaultProblem(CodeWarning $exception)
         }
     }
 
+    /**
+     * Get the value of a single PHPCS config key.
+     *
+     * @param string $key The name of the config value.
+     *
+     * @return string
+     */
+    public static function getConfigData(string $key): string
```

Comment:
Why static and why public?

Reply:
Because I didn't know if I wouldn't have to use the function somewhere else, as this was work in progress. As for the static, I forgot to remove it from the function which was inspired by the PHPCompatibility package.